Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Person.h"
+#include "iostream"
+
+Person::Person() {}
+
+Person::~Person() {
+    delete age_;
+}
+
+Person::Person(const std::string& name, int age) {
+    this->name_ = name;
+    this->age_ = new int(age);
+}
+
+int Person::getAge()const{
+    return *age_;
+}
+
+const std::string& Person::getName()const{
+    return name_;
+}
+
+void Person::setName(const std::string& newName){
+    name_ = newName;
+}
+
+void Person::setAge(int newAge){
+    int age = newAge;
+    age_ = new int(age);
+}
+
+void Person::printInfo() const{
+    std::cout << "\nName: " << Person::name_ <<std::endl;
+    std::cout << "Age: " << *Person::age_ <<std::endl;
+}
+
+int Person::combinedAge(Person** personArray, int size){
+     int combined_age = 0;
+     for(int i = 0; i < size; i++){
+        Person* personPointer = personArray[i];
+        combined_age += personPointer->getAge();
+     }
+     return combined_age;
+}
+
+void Person::birthday(Person& x){
+    (*x.age_) ++;
+}
```

Comment:
Generally, you have a new line at the end of the file. [Here's why](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline )